packages/infra/lib/shared/rds.ts

```diff
@@ -2,19 +2,28 @@ import * as cloudwatch from "aws-cdk-lib/aws-cloudwatch";
 import { SnsAction } from "aws-cdk-lib/aws-cloudwatch-actions";
 import * as rds from "aws-cdk-lib/aws-rds";
 import { Construct } from "constructs";
-import { RDSAlarmThresholds } from "../../config/aws/rds";
 import { mbToBytes } from "../shared/util";
-
+import { EnvConfig } from "../../config/env-config";
 const DEFAULT_MIN_LOCAL_STORAGE_MB_ALARM = 10_000;
```

Comment:
Please keep an empty line after imports to help visualize constants, types, etc., as well as keep the auto-organizer free to reorganize those as needed.